site_scons/daos_build.py

```diff
@@ -154,6 +154,7 @@ def library(env, *args, **kwargs):
 def program(env, *args, **kwargs):
     """build Program with relative RPATH"""
     denv = env.Clone()
+    denv.AppendUnique(LINKFLAGS=['-fpic','-pie'])
```

Comment:
You need to fix the pylint warning here.  

Do we also need to add these to CCFLAGS

Reply:
CCFLAGS includes "-fpic" already. I am going to fix the pylint warning soon. Thank you!